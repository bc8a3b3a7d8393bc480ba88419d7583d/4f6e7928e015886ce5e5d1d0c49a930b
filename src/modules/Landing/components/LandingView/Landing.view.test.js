import { testSnapshots } from '../../../../utils/test.util';

import Component from './Landing.view';

describe('Modules/Landing - Landing View', () => {
  const componentProps = {
    onSelectLanguage: jest.fn(),
    selectedLanguage: {
      languageCode: 'vi',
      languageName: 'Tiếng Việt',
      countryCode: 'VN',
      countryName: 'Việt Nam',
    },
  };

  testSnapshots(Component, [
    {
      description: 'basic render',
      props: componentProps,
    },
  ]);
});
