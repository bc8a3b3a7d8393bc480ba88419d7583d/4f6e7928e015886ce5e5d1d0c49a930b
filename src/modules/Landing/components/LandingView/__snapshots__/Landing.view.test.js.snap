// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modules/Landing - Landing View snapshots basic render 1`] = `
<Component
  style={
    Object {
      "flex": 1,
    }
  }
>
  <ScrollView
    contentContainerStyle={
      Object {
        "padding": 10,
      }
    }
    style={
      Object {
        "backgroundColor": "#2B9FDC",
      }
    }
  >
    <Header
      countryCode="VN"
      countryName="Việt Nam"
      languageCode="vi"
      languageName="Tiếng Việt"
    />
    <LanguageList
      countryCode="VN"
      languageCode="vi"
      onSelectLanguage={[MockFunction]}
    />
  </ScrollView>
  <Footer />
</Component>
`;
